lambda/main.go

```diff
@@ -146,3 +157,29 @@ func invalidateCloudFrontCache(session *session.Session) error {
 
 	return err
 }
+
+func sendSlackNotification(status, message string, isFailure bool) {
+	if SLACK_WEBHOOK_URL == "" {
+		log.Println("SLACK_WEBHOOK_URL not set. Slack notification will not be sent.")
+		return
+	}
+
+	var color string
```

Comment:
```
	var color string
	if isFailure {
		color = "#FF0000" // Red color for failure
	} else {
		color = "#36a64f" // Green color for success
	}
```

This can be changed to 
```
	color :=  "#36a64f"
	if isFailure {
		color = "#FF0000" // Red color for failure
	}
```